install/kubernetes/cilium/templates/cilium-secrets-namespace.yaml

```
@@ -11,4 +11,9 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ $name | quote }}
+  labels:
+    app.kubernetes.io/part-of: cilium
+    {{- with .Values.commonLabels }}
```

This is required because you are already inside a loop, which redefines the root `.` object (it should be one of the causes of the CI failures).

```suggestion
    {{- with $.Values.commonLabels }}
```